main.go

```diff
@@ -33,55 +39,62 @@ func connectionChecker(peer smtpd.Peer) error {
 		}
 	}
 
-	return smtpd.Error{Code: 421, Message: "Denied"}
+	log.Printf("IP out of allowed network range, peerIP:[%s]", peerIP)
+	return smtpd.Error{Code: 421, Message: "Denied - IP out of allowed network range"}
 }
 
 func senderChecker(peer smtpd.Peer, addr string) error {
+	if *allowedSender == "" {
+		// disable sender check, allow anyone to send mail
+		return nil
+	}
```

Comment:
I think the motivation for this was more correctly handled in upstream decke/smtprelay#7.